python/common/org/python/types/Bytes.java

```diff
@@ -555,8 +559,23 @@ public boolean __setattr_null(java.lang.String name, org.python.Object value) {
     @org.python.Method(
             __doc__ = ""
     )
-    public org.python.Object capitalize(java.util.List<org.python.Object> args, java.util.Map<java.lang.String, org.python.Object> kwargs, java.util.List<org.python.Object> default_args, java.util.Map<java.lang.String, org.python.Object> default_kwargs) {
-        throw new org.python.exceptions.NotImplementedError("bytes.capitalize has not been implemented.");
+    public org.python.Object capitalize() {
+        byte[] value = new byte[this.value.length];
+        for (int i = 0; i < this.value.length; i++) {
+            byte b = this.value[i];
+            if (b < 128) { // TODO: double check if this is sufficient safeguard for ascii
```

Comment:
This check should probably be < 127 - ASCII 127 is DEL, which generally isn't printable. You should also have a guard at the "low" end - `>= 32`.